feature_engine/outliers/artbitrary.py

```diff
@@ -91,6 +91,36 @@ class ArbitraryOutlierCapper(BaseOutlier):
     transform:
         Cap the variables.
 
+    Examples
+    --------
+
+    >>> import pandas as pd
+    >>> import numpy as np
+    >>> from feature_engine.outliers import ArbitraryOutlierCapper
+    >>> np.random.seed(42)
+    >>> X = pd.DataFrame(dict(x1 = np.random.exponential(size = 100),
+    >>>                     x2 = np.random.randn(100)))
```

Comment:
Since we are using the arbitrary capper, I think it is more illustrative if we create a toy dataset with values 1,2,3,4... to 10. And when cap at 2 and 8. And show the transformed dataframe. Could we change the example?